contracts/src/Gateway.sol

```diff
@@ -260,7 +262,7 @@ contract Gateway is IGateway, IInitializable {
             revert AgentAlreadyCreated();
         }
 
-        address payable agent = payable(new Agent(params.agentID));
+        address payable agent = payable(new Agent(params.agentID,params.paraID));
```

Comment:
```suggestion
        address payable agent = payable(new Agent(params.agentID, params.paraID));
```

(nit)

Reply:
Revert the change here for https://linear.app/snowfork/issue/SNO-615#comment-48a711d1